src/html5/parser/document.rs

```diff
@@ -457,6 +472,44 @@ impl Document {
 
         false
     }
+
+
+    pub fn add_useragent_stylesheet(&mut self, stylesheet: CssNode) {
+        self.useragent_stylesheets.push(stylesheet);
+    }
+}
+
+// Walk the document tree with the given visitor
+pub fn visit(doc: &DocumentHandle, visitor: &mut Box<dyn Visitor<Node>>) {
+    let binding = doc.get();
+    let root = binding.get_root();
```

Comment:
```suggestion
    let root = doc.get();
    let root = root.get_root();
```

I won't make more comments than this one on the `binding` variables, it's better to rename them, however you can leave it like this if you want.